df_py/util/dftool_module.py

```diff
@@ -1084,6 +1084,41 @@ def do_dispense_passive():
     print("Dispensed passive rewards")
 
 
+# ========================================================================
+
+
+@enforce_types
+def do_dispense_predictoor():
```

Comment:
Similar to the github actions filenames, we need better clarity on these method names.

It should be something like:
- do_dispense_passive
- do_dispense_volume (includes challenge, for now)
- do_dispense_predictoor_ocean
- do_dispense_predictoor_rose

OR
- do_dispense_passive
- do_dispense_active, which dispenses volume, challenge, predictoor ocean, and predictoor rose

Reply:
Thanks for the info. 

OK we don't need `do_dispense_predictoor_rose`. Not worth it.

It seems that `do_dispense_predictoor_ocean` needs a better label, since it "Funds the account which will to bridge OCEAN to Sapphire" versus actually dispensing rewards.